AppleALC/kern_alc.cpp

```diff
@@ -798,7 +813,25 @@ IOReturn AlcEnabler::initializePinConfig(IOService *hdaCodec, IOService *configD
 			}
 		}
 	}
+}
+
+IOReturn AlcEnabler::initializePinConfigTiger(IOService *hdaCodec) {
+	auto parentDevice = hdaCodec->getParentEntry(gIOServicePlane);
+	while (parentDevice) {
+		auto name = parentDevice->getName();
+		if (name && (!strcmp(name, "AppleHDAController"))) {
+			break;
+		}
 
+		parentDevice = parentDevice->getParentEntry(gIOServicePlane);
+	}
+
+	callbackAlc->patchPinConfig(hdaCodec, parentDevice);
```

Comment:
Check parentDevice for NULL here.

Reply:
Fixed in https://github.com/acidanthera/AppleALC/commit/b1047c1a3323c4759b9fc1ee5392ba20bc26b75d.